igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsConfig.groovy

```diff
@@ -40,11 +43,14 @@ import javax.validation.Valid
 @CompileStatic
 class JenkinsConfig {
 
+    @Value('${client.timeout:30}')
```

Comment:
I see below where it is explicitly seconds

Maybe change the config property to client.readTimeoutSeconds